uievents/order-of-events/mouse-events/click-on-div-manual.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Clicking on a div element that has no click event handler fires a click event</title>
+    <link rel="author" title="Chris Rebert" href="http://chrisrebert.com">
+    <link rel="help" href="https://w3c.github.io/uievents/#event-type-click">
+    <link rel="help" href="https://dom.spec.whatwg.org/#concept-event-dispatch">
+    <link rel="help" href="https://www.w3.org/TR/dom/#concept-event-dispatch">
+    <meta name="flags" content="interact">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+#square {
+  width: 100px;
+  height: 100px;
+  background-color: blue;
+  color: white;
+  /* Center the "Click me" text */
+  line-height: 100px;
+  text-align: center;
+}
+    </style>
+  </head>
+  <body>
+    <p>Click on the blue square below. If a "PASS" result appears, the test passes; otherwise, it fails.</p>
+    <div id="square">Click me</div>
+    <script>
+setup({explicit_timeout: true});
+async_test(function(t) {
+  document.addEventListener('click', function (event) {
+    var square = document.getElementById('square');
+    t.step(function () {
+      assert_equals(event.target, square, 'target of click event must be the test square');
+      assert_equals(event.eventPhase, Event.BUBBLING_PHASE, 'click event must propagate to the document object during the Bubbling Phase');
+      var passed = event.target === square && event.eventPhase === Event.BUBBLING_PHASE;
+      square.textContent = passed ? 'PASS' : 'FAIL';
+      square.style.backgroundColor = passed ? 'green' : 'red';
+      t.done();
+    });
+  }, false);
+}, "Clicking on a div element that has no click event handler on the element itself should still fire a click event that ultimately propagates to the document object");
```

Comment:
IIRC the `window` is also part of the tree for the purpose of event dispatch, though I don't find right now where this is defined. But this test doesn't need to test that necessarily, but maybe the test name could be more correct.

In general I would recommend avoiding putting the pass condition in the test name itself. Having the test name only describe what the test is trying to do, and not say what the expected result is, means that the test name is stable even if the spec changes what the pass condition should be.